modal/modal.css

```diff
@@ -0,0 +1,42 @@
+@import url(/common.css);
+@import url(/button/button.css);
+@import url(/card/card.css);
+
+.modal-container{
+    display: none; 
+    position: fixed; 
+    z-index: 1; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(0,0,0);
```

Comment:
You dont require because you are cascading the background-color in next line.

Reply:
oke i wrote that as a fallback value